app/schemas.py

```diff
@@ -209,6 +197,16 @@ class Meta:
         exclude = ("created_at",)
 
 
+class EmailDataSchema(ma.Schema):
+    email = fields.Str(required=False)
+
+    @validates('email')
+    def validate_email(self, value):
+        try:
+            validate_email_address(value)
+        except InvalidEmailError:
+            raise ValidationError('Invalid email')
```

Comment:
It might be better to pass the message form the exception InvalidEmailError so that  the returned response puts the more specific reason in the response json.